src/renderer/components/caption-studio/TextEditor.vue

```diff
@@ -154,6 +154,7 @@ export default {
     EventBus.$on('caption_reset', this.reset);
     EventBus.$on('json_errors', this.onJsonErrors);
     this.$refs.Quill?.quill?.on('text-change', this.onEdit);
+    document.getElementById('quill-editor').addEventListener('paste', catchPasteEvent);
```

Comment:
can you use the `$ref` here instead of the get `elementById`? Its the more vue-ish way to do it